masterlist.yaml

```diff
@@ -185,6 +185,7 @@ globals:
  
 
 plugins:
+### Official Update & DLC plugins
```

Comment:
Same separators were in F04 master-list. Thought it would useful for when groups are added.

Reply:
Groups are already in the `v0.13` branch. I think you might be misunderstanding how they're used though? They're not like BOSS groups where they divide the masterlist up into sections, I anticipate they'll be used much more sparingly (they are just replacing priorities after all).

Reply:
Ah right, for some reason I thought they would be used as the default way to sort the load order with load after overriding them.